sdk/eventhub/event-hubs/samples/v5-beta/javascript/iothubConnectionString.js

```diff
@@ -128,6 +133,12 @@ async function convertIotHubToEventHubsConnectionString(connectionString) {
 async function main() {
   console.log(`Running iothubConnectionString sample`);
 
+  const kvClient = new SecretClient(keyvaultUri, new DefaultAzureCredential());
```

Comment:
samples need to be simpler and with minimum setup, this is one more obstacle and may not be the best for the new users trying out.

Reply:
This is an advanced sample and is not meant to be a starter sample in any way. Using a key vault here is the quite literally the simplest part of it. It accesses an Event Hub instance managed by an IoT Hub resource by sending a bare AMQP request to force an error response and then parses that response to re-construct the connection string of that managed Event Hubs instance.

All simple Event Hubs samples don't manipulate secrets, so they don't need any additional protections.